apps/web/src/config/constants/contracts.ts

```diff
@@ -166,6 +166,10 @@ export default {
     [ChainId.BSC]: '0x695170faE243147b3bEB4C43AA8DE5DcD9202752',
     [ChainId.BSC_TESTNET]: '0x56666300A1E25624489b661f3C6c456c159a109a',
   },
+  bCakeFarmBoosterVeCake: {
```

Comment:
Farm booster seems under the scope of farm to me. Can we move the contract configuration to farms package?

Reply:
hmm or it's actually part of veCake?

Reply:
let me check with SC developer

Reply:
@chefjackson 
The farm booster is based on MCV3, but the multiplier is calculated based on VECake data.
Hmm🤔 moving to one of those should be okay. WDYT

Reply:
move to farms then